NAMESPACE

```diff
@@ -30,43 +30,18 @@ S3method(select,SummarizedExperiment)
 S3method(separate,SummarizedExperiment)
 S3method(slice,SummarizedExperiment)
 S3method(summarise,SummarizedExperiment)
+S3method(summarize,SummarizedExperiment)
 S3method(tbl_format_header,tidySummarizedExperiment)
 S3method(tidy,RangedSummarizedExperiment)
 S3method(tidy,SummarizedExperiment)
 S3method(unite,SummarizedExperiment)
 S3method(unnest,tidySummarizedExperiment_nested)
 export("%>%")
-export()
-export(as_tibble)
-export(bind_rows)
-export(count)
-export(distinct)
-export(extract)
-export(filter)
-export(full_join)
-export(ggplot)
-export(group_by)
-export(inner_join)
-export(left_join)
-export(mutate)
-export(nest)
-export(pivot_longer)
-export(pivot_wider)
+export(bind_cols.RangedSummarizedExperiment)
+export(bind_cols.SummarizedExperiment)
+export(bind_rows.SummarizedExperiment)
```

Comment:
Please have a look, if these are present in `tidySingleCellExperiment` as well, because they look a bit strange

Reply:
I believe it should look like

https://github.com/stemangiola/tidySingleCellExperiment/blob/a3dc67e2c72b6eacd954b202d9fcb61fc246b0b1/NAMESPACE#L6

Reply:
Hello, thank you very much for the prompt reply. I apologize for the commit without the comment.

The warnings above (count, filter, rename, slice) were solved by moving `dplyr` to Depends (https://stat.ethz.ch/pipermail/r-devel/2014-June/069247.html), however, in that way, the other warnings that methods lookup cannot find `bind_cols` and `bind_rows` occurred as below. The additional `@export` was the intention to avoid these warnings.

```
W  checking S3 generic/method consistency (5.7s)
   Warning: declared S3 method 'bind_cols.RangedSummarizedExperiment' not found
   Warning: declared S3 method 'bind_cols.SummarizedExperiment' not found
   Warning: declared S3 method 'bind_rows.SummarizedExperiment' not found
   See section 'Generic functions and methods' in the 'Writing R
   Extensions' manual.
```

`tidyseurat` also has these warnings when checked in R 4.2.3, and `tidySingleCellExperiment` also has them as below if I explicitly change R version dependency and check in R 4.2.3, although these warnings will not occur as `tidySingleCellExperiment` depends on R >= 4.3.0.

```  
W  checking S3 generic/method consistency (7.5s)
   Warning: declared S3 method 'bind_cols.SingleCellExperiment' not found
   Warning: declared S3 method 'bind_rows.SingleCellExperiment' not found
   See section 'Generic functions and methods' in the 'Writing R
   Extensions' manual.
```

Reply:
I would say, let's depend on anything `tidySCE` depends on, and let's set up the github action from `tidySCE` template.

Basically whatever `tidySCE` does we do as well.

Reply:
I apologize that no checks have been made on `BiocCheck` in this PR. Will remove at least errors and warnings.

Reply:
Thanks! Just added a runnable example in `plotly.Rd`. The commit locally passed `BiocCheck` (in `3.18`) without errors (warnings and notes remained, also in `tidySCE`). R CMD CHECK (version `4.3.1`) also passes without errors, warnings, and notes, which should pass GitHub actions.